composer.json

```diff
@@ -51,5 +52,7 @@
                 "Tightenco\\Quicksand\\QuicksandServiceProvider"
             ]
         }
-    }
+    },
+    "minimum-stability": "dev",
```

Comment:
@bakerkretzmar Would you mind just dropping this, I'll merge it, and then we'll tag as soon as Laravel releases 8?